app/models/v2/test_result.rb

```diff
@@ -7,7 +7,13 @@ class TestResult < ApplicationRecord
     self.table_name = :v2_test_results
     self.primary_key = :id
 
-    belongs_to :system, class_name: 'V2::System', optional: true
+    if Rails.env.test? # For testing taggable
```

Comment:
@LightOfHeaven1994 This should probably be removed

Reply:
@romanblanco we did it on purpose as we don't want to set autosave and delegate thing for prod, this is needed for testing only to make a trick

Reply:
Makes sense. Thanks!